ED/build/make/include.mk.arch_linux

```diff
@@ -41,29 +41,40 @@ USE_INTERF=1
 USE_MPIWTIME=1
 
 # Compile flags ------------------------------------------------
-
 CMACH=PC_LINUX1
-
 F_COMP=mpif90
-#F_COMP = gfortran
-
-F_OPTS= -g -Wall -W -ffpe-trap=invalid,zero,overflow -Wconversion -fbounds-check -fbacktrace -fdump-core
-#F_OPTS= -V -FR -O2 -recursive -static -Vaxlib  -check all -g -fpe0 -ftz  -debug extended \
-#        -debug inline_debug_info -debug-parameters all -traceback -ftrapuv
-#F_Opts= -03
-
 C_COMP=mpicc
-
-#C_OPTS= -O2 -DLITTLE  -g -static -traceback -debug extended
-C_OPTS = -O3 -DLITTLE
-
 LOADER=mpif90
+#F_COMP = gfortran
+#C_COMP = gcc
 #LOADER = gfortran
 
-LOADER_OPTS=${F_OPTS}
+##################################### COMPILER OPTIONS #####################################
+#------------------------------------------------------------------------------------------#
+# A/B/C/D. Debugging, strictest compilation flags, lowest performance.                     #
```

Comment:
Probably should add a note that these options compile with mpi turned on while the E doesn't.

Reply:
I simply followed what was already in the original files, but I'm wondering whether these differences are intentional, or just inconsistencies because the dbg was not up-to-date. In case it's the former, I can add the comment.

Reply:
I suspect it's just inconsistencies because one or the other was not up to date, but I'm not sure.